lib/serialisers/json-0.6.js

```diff
@@ -156,9 +161,11 @@ module.exports = createClass({
         }
 
         element.attributes.set('samples', samples);
+      } else {
+        element.content = undefined;
```

Comment:
Without this, during this branch we would leave the enumerations content also inside content. Tested in `enum element deserialises content`.